src/libraries/System.Security.Cryptography/tests/CryptoConfigTests.cs

```diff
@@ -34,6 +34,7 @@ public static void NamedCreateMethods_NullInput()
         // The returned types on .NET Framework can differ when the machine is in FIPS mode.
         // So check hash algorithms via a more complicated manner.
         [Theory]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/61099", TestPlatforms.Browser)]
```

Comment:
I think we should call all of these https://github.com/dotnet/runtime/issues/37669